### ISSUE TITLE
let me turn it on and off with a command in command palette

### ISSUE THREAD

**Issue by aziz** [possan/sublime_unicode_nbsp#1]
it's an awesome plugin. but sometimes I would like to turn it off.

**Comment by possan** [comment]
Yes im suspecting i need to fix that... it seems to slow down on some files

**Comment by Kos** [comment]
Table Editor plugin has commands "enable for current view" and "enable for current syntax", that's a nice way to do it.
However, Package Control's "Enable/Disable Package" works just as well for me.
